src/index.test.js

```diff
@@ -107,6 +107,13 @@ test('notifies on action', t => {
   t.true(handler.calledWithExactly(node0, action))
 })
 
+test('notifies on action without onAction handler', t => {
+  const { tree } = t.context
+  const wrapper = mount(<DropdownTreeSelect id={dropdownId} data={tree} showDropdown="initial" />)
+  wrapper.find('i.fa-ban').simulate('click')
+  t.pass()
```

Comment:
`t.pass`?

Reply:
I'm not entirely sure what I need to check here :thinking: 
The test is only checking if the click event does not trigger error, therefore it pass.

Any suggestions what would be correct test for it otherwise?

Reply:
Well, there is literally no checks/assertions in this test. It adds no value. I'm not sure if there's a way to check for those type of warnings during a test run. If we can't find a way to ensure that a specific warning is not emitted, then I'd rather not have this test.

Reply:
I think one way we could check for this is spy console.warn or console.error (whichever is emitted by React) and then check that it doesn't get called. 

You can remove your default prop and this test should fail - which would confirm that this approach works :)

Reply:
Well, because it's actual JavaScript TypeError, nothing to do with React itself, it already fails by default without any checks, if default prop is removed, like this:

```javascipt
index › notifies on action without onAction handler

....../javascript/react-dropdown-tree-select/src/index.js:216

   215:   onAction = (nodeId, action) => {                                   
   216:     this.props.onAction(this.treeManager.getNodeById(nodeId), action)
   217:   }                                                                  

  Error thrown in test:

  TypeError {
    message: '_this.props.onAction is not a function',
  }

```

I changed from pass() to spy for console error, but honestly I don't see any value in it.

Let me know if this acceptable now

Reply:
Well `pass` is of no value. This test actually checks and will warn in future if we break it again. I'll check it and merge (mostly looks good).